www/md_base/src/app/home/home.route.coffee

```diff
@@ -17,3 +17,24 @@ class State extends Config
             templateUrl: "views/#{name}.html"
             name: name
             url: '/'
+
+        bbSettingsServiceProvider.addSettingsGroup
+            name: 'home'
+            caption: 'Home'
+            description: 'Settings to customize the display of Home page'
+            items: [
```

Comment:
That is fine enough. Eventually, I would have liked a plugin system, where people could have developed their own widget for the home page.